packages/git-unix/git-unix.3.1.0/opam

```diff
@@ -0,0 +1,64 @@
+opam-version: "2.0"
+synopsis: "Virtual package to install and configure ocaml-git's Unix backend"
+maintainer: ["thomas@gazagnaire.org" "romain.calascibetta@gmail.com"]
+authors: "Thomas Gazagnaire"
+license: "ISC"
+homepage: "https://github.com/mirage/ocaml-git"
+doc: "https://mirage.github.io/ocaml-git/"
+bug-reports: "https://github.com/mirage/ocaml-git/issues"
+depends: [
+  "ocaml" {>= "4.07.0"}
+  "dune" {>= "2.6.0"}
+  "mmap" {>= "1.1.0"}
+  "stdlib-shims"
+  "git" {= version}
+  "rresult"
+  "result"
+  "bigarray-compat"
+  "bigstringaf"
+  "fmt" {>= "0.8.7"}
+  "bos"
+  "fpath"
+  "uri" {with-test}
+  "digestif" {>= "0.8.1"}
+  "logs"
+  "lwt"
+  "base-unix"
+  "alcotest" {with-test & >= "1.1.0"}
+  "alcotest-lwt" {with-test & >= "1.1.0"}
+  "base64" {with-test & >= "3.0.0"}
+  "git-cohttp-unix" {= version}
+  "mirage-clock"
+  "mirage-clock-unix"
+  "astring" {>= "0.8.5"}
+  "awa"
+  "cmdliner" {>= "1.0.4"}
+  "cohttp-lwt-unix" {>= "2.5.4"}
+  "decompress" {>= "1.2.0"}
+  "domain-name" {>= "0.3.0"}
+  "ipaddr" {>= "5.0.1"}
+  "mtime" {>= "1.2.0"}
+  "ocamlfind" {>= "1.8.1"}
```

Comment:
Does `git-unix` really need ocamlfind?

Reply:
I think it's needed for `git-unix.top` which is a simple REPL layer to load pretty-printers and libraries (`ocamlfind`) required.

Reply:
I'm not very convince that this part is useful for the end-user but it's definitely good for the developer (who can introspect `git` with `dune buiild && ocaml`). If you think that we should remove this part, I can redo a release without it.

Reply:
Not at all, I was just checking it hadn't snuck in by mistake. Thanks for the clarification :-)